specs/latest/2.0/index.html

```diff
@@ -1273,7 +1273,11 @@ <h2 class="no-toc">Table of contents</h2>
         Transfer a rectangle of pixel values from one region of the read framebuffer to another in the draw
         framebuffer. If the value of SAMPLE_BUFFERS for the read framebuffer is one and the value of
         SAMPLE_BUFFERS for the draw framebuffer is zero, the samples corresponding to each pixel location in
-        the source are converted to a single sample before being written to the destination.
+        the source are converted to a single sample before being written to the destination. <br><br>
+
+        If the source and destination buffer are identical, which means they have the same image with the same mipmap level
+        of texture and the same layer of 3D or 2D_ARRAY texture and the same face of CUBE_MAP texture,
+        generates an <code>INVALID_OPERATION</code> error.
```

Comment:
I don't think this is necessary, because there is no deviation from the ES3 spec.